CMakeLists.txt

```diff
@@ -73,7 +74,15 @@ else()
   message(STATUS "Unrecognized build type - will use cmake defaults")
 endif()
 
-if(ENABLE_CCACHE AND (CMAKE_CXX_COMPILER_ID MATCHES "Clang" OR CMAKE_CXX_COMPILER_ID MATCHES "GNU"))
+# Prefer sccache if it's enabled - it makes little sense to support both.
+# It is defaulted to OFF, and ccache is better for local builds
+if(ENABLE_SCCACHE AND (CMAKE_CXX_COMPILER_ID MATCHES "Clang" OR CMAKE_CXX_COMPILER_ID MATCHES "GNU"))
+  find_program(SCCACHE_FOUND sccache)
+  if(SCCACHE_FOUND)
```

Comment:
Should we error out if user enabled `sccache` but the executable was not found? Currently it will silently switch to running without it.